app/models/manageiq/providers/openstack/infra_manager/host/operations.rb

```diff
@@ -0,0 +1,24 @@
+module ManageIQ::Providers::Openstack::InfraManager::Host::Operations
+  def self.included(base)
+    base.send :include, InstanceMethods
+    base.extend ClassMethods
+  end
+
+  module InstanceMethods
+    def ironic_fog_node
+      fog_baremetal = ext_management_system.openstack_handle.baremetal_service
+      fog_baremetal.nodes.find_by_uuid(uid_ems)
+    end
+
+    def ironic_fog_set_node_maintenance
```

Comment:
Makes sense. I was following the same naming approach used in https://github.com/ManageIQ/manageiq/pull/6996 -- so if I shorten the api calls here, that PR possibly needs a similar change.